test/services/licence-monitoring-station/setup/full-condition.service.test.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const Sinon = require('sinon')
+
+const { describe, it, beforeEach, afterEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const LicenceHelper = require('../../../support/helpers/licence.helper.js')
+const LicenceVersionHelper = require('../../../support/helpers/licence-version.helper.js')
+const LicenceVersionPurposeHelper = require('../../../support/helpers/licence-version-purpose.helper.js')
+const LicenceVersionPurposeConditionHelper = require('../../../support/helpers/licence-version-purpose-condition.helper.js')
+const LicenceVersionPurposeConditionTypeHelper = require('../../../support/helpers/licence-version-purpose-condition-type.helper.js')
+const SessionHelper = require('../../../support/helpers/session.helper.js')
+
+// Thing under test
+const FullConditionService = require('../../../../app/services/licence-monitoring-station/setup/full-condition.service.js')
+
+const CES_CONDITION_TYPE = LicenceVersionPurposeConditionTypeHelper.select(20)
+
+describe('Full Condition Service', () => {
+  let condition
+  let licence
+  let session
+
+  beforeEach(async () => {
+    licence = await LicenceHelper.add()
```

Comment:
One of the reasons why we like to pull any significant fetch queries out of the main services is to make testing the main services simpler as we can just stub the "fetch" services.

For consistency I'd suggest that rather than using the helpers to actually create the test data, that the fetch service is stubbed instead. This is generally how we would do this, here's an example `test/services/monitoring-stations/view.service.test.js`